ivi-layermanagement-api/test/TestBase.cpp

```diff
@@ -151,6 +160,16 @@ TestBase::TestBase()
         throw std::runtime_error("shared memory buffers format ARGB888 is not supported");
     }
 
+    //@Todo need to hard update the checks when changing the version of protocol files
+    if (iviWmCtrlVer != 1)
+    {
+        throw std::runtime_error("Can't find the proper version of ivi_wm protocol on server");
```

Comment:
please the expected and released versions here!
like
Error: ivi_wm protocol version miss match, expected version 1, received version from compositor X

Reply:
I updated the comment, now there are two error cases:
- Server don't have the ivi_wm or ivi_input protocols
- There is an unexpected version on server